echo-web/config/echo.yml

```diff
@@ -5,10 +5,10 @@ spinnaker:
   baseUrl: 'http://localhost:9000'
 
 echo:
-  baseUrl: 'http://localhost:8080'
+  baseUrl: 'http://localhost:8088'
```

Comment:
I would leave this as is, and keep a `-local.yml` profile either in your hal profile list or in ~/.spinnaker